### DISCUSSION TITLE
rtl file export in tailwind

### DISCUSSION THREAD

**Discussion by alimoslem** [tailwindlabs/tailwindcss#7828] [Help] [answered]
how can I use rtl in tailwind??

and how can I export rtl file in tailwind-cli??

**Comment by motahharm** [comment, marked as answer]
Hi @alimoslem 

you can use rtl in class like this:
`<p class="pl-5 rtl:pr-5">بررسی کردن قابلیت راست به چپ</p>`
and html must be rtl:
`<html dir="rtl">`

if you want to export rtl file you will use <a href="https://rtlcss.com/">RTLcss</a>.

**Comment by alimoslem** [comment]
thanks @motahharm.
